engines/agi/op_cmd.cpp

```diff
@@ -810,22 +834,31 @@ void cmdInitDisk(AgiGame *state, AgiEngine *vm, uint8 *parameter) {
 // %v<number>: at this place the output will include a decimal value of variable with the given number.
 // %w<number>: the text of the player - entered word with the given index number is inserted at this place. (The index is 'one based'; i.e.first word is % w1, second is % w2, etc.)
 //
-// 2. Logs not written to a file.
-// At the moment we are just logging to the console. To see
-// the logs, use the following arguments to scummvm:
-//
-// --debugflags=Scripts -d 1
-//
 void cmdLog(AgiGame *state, AgiEngine *vm, uint8 *parameter) {
 	uint16 textNr = parameter[0];
 	if (state->_curLogic->texts != nullptr && (textNr - 1) <= state->_curLogic->numTexts) {
 		byte currentRoom = vm->getVar(VM_VAR_CURRENT_ROOM);
 		const char *inputLine = (char *)vm->_text->_promptPrevious;
 		const char *message = state->_curLogic->texts[textNr - 1];
 
-		debugC(1, kDebugLevelScripts, "Room %hhu", currentRoom);
-		debugC(1, kDebugLevelScripts, "Input line : %s", inputLine);
-		debugC(1, kDebugLevelScripts, "%s", message);
+		Common::String logMessage = Common::String::format("Room %hhu\nInput line : %s\n%s\n",
+														   currentRoom,
+														   inputLine,
+														   message);
+
+		// Logs to the console. To see, use arguments: --debugflags=Scripts -d 1
+		debugCN(1, kDebugLevelScripts, "%s", logMessage.c_str());
+
+		static Common::DumpFile dumpFile;
```

Comment:
I am not a big fan of static objects like this. Why not move it to the `AgiEngine` class variable?